react-components/src/library/components/learner-report-form/index.js

```diff
@@ -17,6 +17,13 @@ const getQueryLimitParam = () => {
   return urlParams.get('queryLimit')
 }
 
+const hasAtLeastTwoAlphanumeric = (str) => {
+  // This regular expression matches alphanumeric characters
+  const matches = str.match(/[a-zA-Z0-9]/g)
```

Comment:
You could update the regex to `/[a-zA-Z0-9]{2,}/g` and then remove the length check :)